crates/forge/bin/cmd/test/mod.rs

```diff
@@ -756,12 +812,23 @@ async fn test(
                 gas_report.analyze(&result.traces);
             }
         }
-        let block_outcome = TestOutcome::new([(contract_name, suite_result)].into(), allow_failure);
+        let block_outcome =
+            TestOutcome::new([(contract_name.clone(), suite_result)].into(), allow_failure);
 
         total_passed += block_outcome.successes().count();
         total_failed += block_outcome.failures().count();
         total_skipped += block_outcome.skips().count();
 
+        if summary {
+            let row = create_test_summary_table_row(
+                contract_name,
```

Comment:
Like @mattsse said, alphabetical sorting here would be nice, it's unclear what the current ordering is